src/components/hamburger/SocLink.tsx

```diff
@@ -0,0 +1,19 @@
+// SocLink.tsx
+type SocLinkProps = {
+  href: string;
+  text: string;
+  icon: string;
+};
+
+export const SocLink: React.FC<SocLinkProps> = ({ href, text, icon }) => {
```

Comment:
let's be consistent here please, use the classic approach we have everywhere `export const Component = ({prop1, prop2, ..}: Props) =>` ...